src/libstd/net/addr.rs

```diff
@@ -705,30 +705,75 @@ impl hash::Hash for SocketAddrV6 {
 ///
 /// # Examples
 ///
+/// Creating a [`SocketAddr`] iterator that yields one item:
+///
+/// ```
+/// use std::net::{ToSocketAddrs, SocketAddr};
+///
+/// let mut addrs_iter = ((127, 0, 0, 1), 443).to_socket_addrs().unwrap();
```

Comment:
Looks like there's not an impl for `((u8, u8, u8, u8), u16)`? Maybe you still need a `Ipv4Addr::new`?